woocommerce/woocommerce-gutenberg-products-block
# Add product search block

Implements the product search block.

Fixes #592



### Screenshots

![Edit Post ‹ local wordpress test — WordPress 2019-07-08 14-16-17](https://user-images.githubusercontent.com/90977/60813151-fa651c80-a18a-11e9-949e-67370e4f1f21.png)

### How to test the changes in this Pull Request:

1. `npm run build`
2. Add Product Search block to a page or post
3. Test editing the label
4. Test editing the placeholder
5. Test turning off the label (toggle in sidebar)
6. Test searching on the frontend.
7. Refresh editor and ensure block settings are preserved



### Changelog

> Add new product search block to add a product search box field to a page.
